source/documentation/other-topics/Setup-postgres-container.md

```diff
@@ -0,0 +1,79 @@
+### Setup Postgres container
+
+If you need to quickly create DB for test/dev, you can set up a pod in your namespace running postgres in a container. Deploying [Bitnami PostgreSQL][postgresql-chart] as Helm Chart is the easiest way to get started with PostgreSQL on Kubernetes. This chart bootstraps a PostgreSQL deployment on a Kubernetes cluster using the Helm package manager. 
+
+The [django-reference-application][django-app] uses Bitnami PostgreSQL Helm Chart to add a postgres instance, you can use the same Chart to setup postgres in your namespace.
+
+> Note: Even though we are going to install a database within the Kubernetes cluster, it is recommended to use a database as a service offering such as [AWS RDS](https://aws.amazon.com/rds/) if running in production.
+
+##### Requirements
+It is assumed you have the following:
+
+ - You have [created an environment for your application][env-create]
+ - You have installed [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) on your local machine.
+ - You have [Authenticated][auth-to-cluster] to the cloud-platform-live-1 cluster.
+ - You have configured [Helm and Tiller][using-helm]. 
+
+##### Set up
+First we need to clone our reference application and change directory:
+
+    $ git clone https://github.com/ministryofjustice/cloud-platform-reference-app.git
+    $ cd cloud-platform-reference-app/helm_deploy/django-app/charts/postgresql
```

Comment:
This doesn't work. There is no "postgresql" directory on that path, only a `postgresql-4.0.1.tgz` file.

Also, is it really necessary to clone the entire reference app repo. if you only want the helm chart? Isn't there a way to just fetch the part of that repo the user actually needs? I think that would be less confusing.

Finally, do they really need to use the reference app. repo at all? If all they want is a standard postgres container, isn't it easier to use a "stable" helm chart (if there is one), or create a deployment using the bitnami postgresql container?

Reply:
Thanks @digitalronin, Sorry there is a PR for it, once it get merged we will have a "postgresql" directory
https://github.com/ministryofjustice/cloud-platform-reference-app/pull/76

Reply:
As now the above changes got merged, updated the user-guide considering the review comments.